Базовая стилизация всех страниц

KuzminaPolina opened the pull request:
1. На главной странице есть много всего помимо базовых стилей. Я сильно подозреваю, что там многое сделано не так, как дальше будет в гайдах академии. Я пока сделала, как умею, потом посмотрю дальше, как сделано в примерах академии, и исправлю. Особенно что касается скрытия заголовков х)
2. На странице каталога реально только базовая стилизация, не считая хэдэра и футера... просто и так уже просрочила сдачу.
3. У меня есть идея, как сделать исчезающий жёлтый фон у главного меню, но я пока не успела проверить, сработает ли мой способ.
4. Жду подробного фидбэка и наверное в ближайшее время надо договориться о консультации голосом, прояснить разные вопросы по стилям.


---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2098917/tasks/8)

:boom: https://htmlacademy-htmlcss.github.io/2098917-device-35/4/

KuzminaPolina committed:
added styles for index page

This is a "save" push

KuzminaPolina committed:
added styles for index and catalog

senchkim commented on styles/style.css:
```diff
@@ -0,0 +1,809 @@
+.shop-menu-link {
+  text-decoration: none;
+  height: 60px;
+  width: 160;
```
тут наверное `160px` должно быть

KuzminaPolina replied:
Нет, там и должно быть 60рх, но я теперь поняла, что там в другом косяк... х))

senchkim commented on styles/style.css:
```diff
@@ -0,0 +1,809 @@
+  width: 160px;
+  height: 20px;
+  padding: 9px 120px 11px 0;
+  font-family: 'Raleway';
```
Забыла указать семейство шрифта, это обязательно нужно делать!
Исправить ниже по коду везде, в тэге боди же верно указала)

KuzminaPolina replied:
Вопрос: я в боди указала семейство шрифта, да. Т.е. это получается шрифт "по-умолчанию" для всей страницы, всё содержимое таким шрифтом отображается, если не указано иное. Я думала, что после боди семейство шрифта надо указывать только в тех случаях, когда оно не совпадает со шрифтом в боди? Или всё-таки вообще везде надо всё равно указывать?

senchkim replied:
Смотри у тебя 2 варианта:
либо добавляешь только для боди шрифт и он далее наследуется!
либо если указываешь шрифт то и семейство тоже нужно указывать!

senchkim commented on styles/style.css:
```diff
@@ -0,0 +1,809 @@
+
+.delivery-partners-item::before {
+  content: "\2022";
+  color: #FFC700;
```
на строке `367` к примеру, цвет указан маленькими буквами, а тут большими - нужно сделать единообразно везде, либо с большими либо с маленькими

senchkim commented on styles/style.css:
```diff
@@ -0,0 +1,809 @@
+  margin: 0 auto;
+}
+
+.about-shop h2 {
```
`h2` тэг лучше заменить на класс, я понимаю что он даже вложенный. но лучше все же по классу стилизовать 👍

senchkim commented on styles/style.css:
```diff
@@ -0,0 +1,809 @@
+}
+
+.info-icon {
+  padding: 30px 42.5px 30px 42.5px;
```
Паддинги и маргины - лучше все же указывать целыми значениями, округляя их до целого!

KuzminaPolina replied:
а если центровка от этого слетает? Всё равно округлять?

senchkim replied:
да, у нас есть погрешность в 5px - поэтому нормально 👍

senchkim commented on catalog.html:
```diff
@@ -5,83 +5,91 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles/style.css">
+
```
И еще заметил что у тебя в верстке прописаны названия картинок и иконок которых нет в проекте!
1) `enter.svg`
2) `compare.svg`
3) `vr-glasses.svg`
4) `social-telegram.svg`

KuzminaPolina committed:
making corrections

KuzminaPolina committed:
making corrections in images

KuzminaPolina committed:
adding final corrections

Overview:
1. Shop menu in yellow boxes corrected;
2. Font-family added where necessary;
3. Hex codes corrected;
4. Hidden headings classes corrected;
5. Fractional numbers replaced.
